data/json/bionics.json

```diff
@@ -139,23 +140,23 @@
     "type": "bionic",
     "name": { "str": "Blood Analysis" },
     "description": "Small sensors have been surgically implanted in your heart, allowing you to analyze your blood.  This will detect many illnesses, drugs, and other conditions.",
-    "occupied_bodyparts": [ [ "torso", 5 ] ],
+    "occupied_bodyparts": [ [ "torso", 4 ] ],
     "act_cost": "100 J"
   },
   {
     "id": "bio_blood_filter",
     "type": "bionic",
     "name": { "str": "Blood Filter" },
     "description": "A filtration system in your heart allows you to actively filter out chemical impurities, primarily drugs.  It will have limited impact on viruses.  Note that it is not a targeted filter; ALL drugs in your system will be affected.",
```

Comment:
```suggestion
    "description": "A filtration system near your heart allows you to actively filter out chemical impurities, primarily drugs.  It will have limited impact on viruses.  Note that it is not a targeted filter; ALL drugs in your system will be affected.",
```